### ISSUE TITLE
MS Edge with Application Guard does not work steam

### ISSUE THREAD

**Issue by JonasSi** [digitalcreations/MaxTo#288]
I'm evaluating MaxTo for work. We use Microsoft Edge with Application Guard (Application Guard is MS way of sandboxing web browsing).
I can't get that window to maximize in any "virtual monitors". It only maximizes to the whole screen. Is there something i'm missing or could this be the same issue as with the Steam client? (https://github.com/digitalcreations/MaxTo/issues/54)

I tried the latest stable release 2017.07

**Comment by vegardlarsen** [comment]
Thank you for reporting this. I have tried to read what I can find about Application Guard, and tried it out my self. You are correct; these windows appear to be untouchable (I tried with the 2.0.0 betas). It is different in behavior from the Steam client though.

I don't know exactly the mechanism it uses, but Application Guard runs the edge browser in a virtual machine, and it may be that it does not forward the window messages correctly into the virtual machine.

I am going to leave this issue open as a confirmed bug; but I don't expect I will be able to do much about this.
